scrapy/link.py

```diff
@@ -11,9 +11,9 @@ class Link:
 
     Using the anchor tag sample below to illustrate the parameters::
 
-            <a href="/nofollow.html#foo" rel="nofollow">Dont follow this one</a>
+            <a href="https://example.com/nofollow.html#foo" rel="nofollow">Dont follow this one</a>
```

Comment:
I actually think that it’s good that the URL was not absolute here. We simply need to indicate, either in the preceding paragraph or in the documentation about the `url` parameter below, that the example `url` value assumes that this code comes from `https://example.com`.

Directly making this an absolute URL in the code example hides the fact that the `url` parameter in `Link` must be an absolute URL.

Although it could be argued that saying `absolute URL` instead of `address` below should be enough to clarify that. I guess keeping the absolute URL here but indicating that below would work for me as well.

Reply:
True. I've modified it.